pkg/datastore/datastore.go

```diff
@@ -217,19 +230,19 @@ func (ds *memoryDatastore) GetPackageIDsByOperatorSource(opsrcUID types.UID) str
 	return strings.Join(packages, ",")
 }
 
-func (ds *memoryDatastore) CheckPackages(packageIDs []string) error {
+func (ds *memoryDatastore) CheckPackages(source string, packageIDs []string) error {
 	missingPackages := []string{}
 	for _, packageID := range packageIDs {
-		if _, err := ds.Read(packageID); err != nil {
+		if _, err := ds.Read(source, packageID); err != nil {
 			missingPackages = append(missingPackages, packageID)
 			continue
 		}
 	}
 
 	if len(missingPackages) > 0 {
 		return fmt.Errorf(
-			"Still resolving package(s) - %s. Please make sure these are valid packages.",
-			strings.Join(missingPackages, ","),
+			"Still resolving package(s) - %s. Please make sure these are valid packages within the %s spec.Source.",
```

Comment:
What is the point is saying `spec.Source`? You have documented it to be an `OperatorSource`

Reply:
I had updated this from `source` after the conversation [here](https://github.com/operator-framework/operator-marketplace/pull/197#discussion_r294982432) but I think your suggestion is better. I will update the message to:

> "Still resolving package(s) - %s. Please make sure these are valid packages within the %s OperatorSource.",